### ISSUE TITLE
NamingSystem for identifiers

### ISSUE THREAD

**Issue by alexmbennett2** [kind-lab/mimic-fhir#71]
Currently, any identifiers in mimic-fhir reference a custom URLs such as http://mimic.fhir.mit.edu/identifier/patient, but nothing is defined for that. It was brought up that a NamingSystem could make sense here. Will look into this further!

**Comment by alexmbennett2** [comment]
@jpwiedekopf any experience with NamingSystems?

**Comment by jpwiedekopf** [comment]
Not really, but the resource is actually designed for that purpose: it defines a namespace for identifiers independent of its usage in the resource, thus adding metadata to the canonical URL of the identifier. I think they would look somewhat like this example from the FHIR spec for us: https://hl7.org/fhir/namingsystem-example-id.json.html

**Comment by alexmbennett2** [comment]
Okay ya that makes sense. Once the mimic-ed branch is wrapped up I'll look to update the identifier links to NamingSystems
